MediaBrowser.Providers/Plugins/Omdb/OmdbProvider.cs

```diff
@@ -459,74 +469,104 @@ private bool IsConfiguredForEnglish(BaseItem item)
             return string.Equals(lang, "en", StringComparison.OrdinalIgnoreCase);
         }
 
-        internal class SeasonRootObject
+        public class SeasonRootObject
         {
+            [JsonConverter(typeof(JsonOmdbNotAvailableStringConverter))]
             public string Title { get; set; }
 
+            [JsonConverter(typeof(JsonOmdbNotAvailableStringConverter))]
             public string seriesID { get; set; }
 
-            public int Season { get; set; }
+            [JsonConverter(typeof(JsonOmdbNotAvailableStructConverter<int>))]
+            public int? Season { get; set; }
 
+            [JsonConverter(typeof(JsonOmdbNotAvailableStructConverter<int>))]
             public int? totalSeasons { get; set; }
 
             public RootObject[] Episodes { get; set; }
 
+            [JsonConverter(typeof(JsonOmdbNotAvailableStringConverter))]
             public string Response { get; set; }
         }
 
-        internal class RootObject
+        public class RootObject
         {
+            [JsonConverter(typeof(JsonOmdbNotAvailableStringConverter))]
```

Comment:
Isn't creating a new `JsonSerializerOptions` and adding `JsonOmdbNotAvailableStringConverter` a cleaner solution?

Reply:
Looks definitely cleaner. But maybe we should use the attributes in case this object is deserialized somewhere else and the converter isn't added?

Reply:
These aren't supposed to get use somewhere else. Not sure why they are public now, but they used to be internal